helm/values.yaml

```diff
@@ -32,4 +32,10 @@ app:
         onecx-shell-footer:
           name: 'onecx-shell-footer'
           description: 'Shell footer'
+        onecx-search-config:
+          name: 'onecx-search-config'
+          description: 'Shell search config'
```

Comment:
Please provide better descriptions. Please add the info that this is a slot in the libs and therefore can be used in all apps and that it is added here to avoid adding it to all apps. Please also say what these slots are used for